debian/changelog

```diff
@@ -1,3 +1,12 @@
+apt-transport-in-toto (0.1.1) unstable; urgency=low
+
+  * Update intoto.py to use `securesystemslib`
+  * debian/control:
+    - Bump standards version to 4.5.1
+    - Add python3-securesystemslib in Depends
```

Comment:
All these things are already part of the initial 0.1.0 release. Right? So I'd probably remove them here.
But we should add the required in-toto 0.3.0 to 1.0.0 min dependency bump, and the potential change in the Uploaders list, both mentioned above. What do you think?

Reply:
Yes agree like this. The current entries are indeed from the original upload. I'll adapt right now with all your comments.

Reply:
Awesome, would you mind if we change this PR title to something like "Prepare 0.1.1 release" and change the base branch from `debian` to `develop`?